database-memory-agent/planning.py

```
@@ -0,0 +1,101 @@
+    return response
+
+def get_llm_response(messages, system_message_content):
+    system_message = {
+        "role": "system",
+        "content": system_message_content,
+    }
+    
+    if any(msg.get("role") == "system" for msg in messages):
+        messages.append(system_message)
+    else:
+        messages = [system_message] + messages
```

Ruff (0.14.8)

Consider `[system_message, *messages]` instead of concatenation

Replace with `[system_message, *messages]`

(RUF005)